.github/workflows/ci-llama-quick-tests.yaml

```diff
@@ -47,6 +47,8 @@ jobs:
         uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
         with:
           python-version: ${{matrix.version}}
+      - name: Create Python venv
+        run: python -m venv ${VENV_DIR}
 
       - name: Cache Pip Packages
```

Comment:
Notes to self:

Need to be deliberate with pip caching. I think I want the downloads to be cached, but _not_ any installed packages.

Reference https://adamj.eu/tech/2023/11/02/github-actions-faster-python-virtual-environments/ and https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#caching-packages.

Both of those approaches hash the requirements files as the cache key, but we have so many separate requirements files _and_ we install our own dev and nightly packages on top, which are not pinned in any requirement files to be cached.

Some of that will hopefully be less consequential, if/when we switch to the `setup_venv.py` + `uv` setup style that I'm prototyping on https://github.com/nod-ai/shark-ai/pull/625.

Reply:
Maybe try landing tomorrow without any pip/venv caching, then figure out how to add it back carefully.  Some extra CI time is worth the environment consistency.